backend/app/assets/javascripts/spree/backend/format_money.js

```diff
@@ -0,0 +1,10 @@
+(function() {
+  var defaultLocale = document.documentElement.lang;
+
+  Spree.formatMoney = function(amount, currency, options) {
```

Comment:
I guess we can both agree that "temporary fix" tends to stay around much longer than what we planned it to stay. :)